test/integration.js

```diff
@@ -11,6 +11,8 @@ const assert = require('assert')
 const PORT = 8282
 const req = getReq(PORT)
 
+const grenacheService = require('../lib/helpers/grenache-service')
```

Comment:
I believe you should not use the real plugin as it tries to connect to a grape

Reply:
ah i see, fixed